db/fixtures/miq_product_features.yml

```diff
@@ -2570,20 +2570,12 @@
     :feature_type: view
     :identifier: tasks_view
     :children:
-    - :name: All VM and Container Analysis Tasks
-      :description: Display Lists of All VM and Container Analysis Tasks
-      :feature_type: view
-      :identifier: job_all_smartproxy
```

Comment:
@isimluk thank you for reviewing! Features removed in this PR were already release, they just become not used after removing screens they control access to.
@Fryguy Do I need to add data migration to this PR to remove not used features from DB or seeding will take care of it?